sleap/nn/training.py

```diff
@@ -1868,14 +1868,9 @@ def main(args: Optional[List] = None):
     parser.add_argument("--prefix", default="", help="Prefix to prepend to run name.")
     parser.add_argument("--suffix", default="", help="Suffix to append to run name.")
 
-    parser.add_argument(
-        "--resume",
-        action="store_true",
-        help=("Resume training from last checkpoint."),
-    )
     parser.add_argument(
         "--base_checkpoint",
-        default="",
+        type=str,
```

Comment:
I think this might still need a default or argparse will complain? You can set it to `default=""` and we can check downstream if it's either `None` or empty, both of which denote that there's no base checkpoint to resume from.

Reply:
Is the unit test `test_resume_training_cli()` written in `test_training.py` sufficient for checking argparse complaining?

Link to relevant line: https://github.com/talmolab/sleap/blob/bd8b2da362b0211b3d8495c7f1436c9be97bd199/tests/nn/test_training.py#L313